### PR TITLE
PSQLADM-418 2.4.3 release notes and release note tasks

### PR DESCRIPTION
**Pull request by patrickbirch**
Added 2.4.3.md

### CODE REVIEW

**Review by georgelorchpercona — APPROVED** [review]

**Review by mohitj1988 — APPROVED** [review]
Looks good

**Review by alina-derkach-oaza — APPROVED** [review]
LGRM

File: docs/2.4.3.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+This release includes all features and bug fixes available in ProxySQL 2.4.3. The improvements and bug fixes introduced by *ProxySQL* and included in this release are:
+
+* Updated to OpenSSL 3.0.5
+* Added an error message, `PMC-10003` about the invalid use of `mysql-init-connect`
+* Fixed the processing of escaped string delimiters for digests
+
+For more information on these items and other improvements and bug fixes, see the *ProxySQL* [2.4.3 release notes](https://github.com/sysown/proxysql/releases/tag/v2.4.3).
+
+*ProxySQL Admin* and *Percona Scheduler Admin* have not changed since the previous release.
```
Comment (venkatesh-prasad-v) [inline docs/2.4.3.md:25]: We fixed the following two bugs for 2.4.3 release.

1. https://jira.percona.com/browse/PSQLADM-402 - Positional Parameters in proxysql-status was not working in previous releases. It has been fixed now.
2. https://jira.percona.com/browse/PSQLADM-419 - Addresses CVE-2021-38561 in pxc_scheduler_handler.

**Review by venkatesh-prasad-v — CHANGES_REQUESTED** [review]

File: docs/2.4.3.md (hunk 1 above)
Comment (patrickbirch) [reply docs/2.4.3.md:25]: Added.

**Review by venkatesh-prasad-v — APPROVED** [review]

**Review by mohitj1988 — APPROVED** [review]

**Pull request merged**